trade-finance-manager-ui/templates/utilisation-reports/add-to-an-existing-payment.njk

```diff
@@ -5,13 +5,20 @@
 {% import "./_macros/fee-record-details-table.njk" as feeRecordDetailsTable %}
 {% import "./_macros/payment-group-input.njk" as paymentGroupInput %}
 {% extends "index.njk" %}
-{% set backLink = {"href": '/utilisation-reports/' + reportId} %}
 {% set pageHeading = "Add reported fee" + ("s" if reportedFeeDetails.feeRecords | length > 1 else "") + " to an existing payment" %}
 {% block pageTitle %}
   {% if errors.errorSummary | length %}Error - {% endif %}
   {{ pageHeading }}
 {% endblock %}
 {% block content %}
+  {{ govukBackLink({
+    text: "Back",
+    href: backLinkHref,
+    attributes: {
+      "id": "back-link",
+      "data-cy": "back-link"
+    }
+  }) }}
```

Comment:
Why can't we use `{% set backLink = {"href": backLinkHref } %}`?

Reply:
Thanks for spotting, this was left over from the old approach where the ID was being used to add an event listener. Replaced with the standard approach you referenced.